cypress/integration/App.spec.js

```diff
@@ -1,4 +1,14 @@
+import { firebase } from '../../config/firebase';
+import 'firebase/auth';
+
+
 describe ('Test App', () => {
+
```

Comment:
Added a before hook to this existing test that will always log an existing user out before testing.